js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1184,7 +1184,7 @@
 
     checkCollectionPermissions: function (collectionID, roCallback) {
       var url = arangoHelper.databaseUrl('/_api/user/' +
-        encodeURIComponent(window.App.userCollection.activeUser) +
+        encodeURIComponent(window.App.userCollection.activeUser || "root") +
```

Comment:
Correct, but we can have authentication disabled but the server being in read-only mode. The problem is that when authentication is disabled we have no user in the frontend. But we still need to make the request to the server because this is the only way we can determine whether we are in read-only mode or not. I also don't like the hack to simply use "root" in case authentication is disabled, but I could not think of a better approach.

Reply:
@mpoeter @ajurdak Ok in that case, my comments / code adjustments can be removed. But please add a TODO / Comment / Jira Card (please decide what suites best) to keep track of that issue. Otherwise we will forget why "root" is being used here. Will approve the PR after the issue is being tracked.